src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IConversionExpression.vb

```diff
@@ -2057,6 +2057,31 @@ BC30512: Option Strict On disallows implicit conversions from 'Integer' to 'SByt
                 additionalOperationTreeVerifier:=AddressOf New ExpectedSymbolVerifier().Verify)
         End Sub
 
+        <Fact()>
+        Public Sub ConversionExpression_Implicit_WideningInterpolatedStringToIFormattable()
+            Dim source = <![CDATA[
+Imports System
+Module Program
+    Sub M1()
+        Dim formattable As IFormattable = $"{"Hello world!"}"'BIND:"Dim formattable As IFormattable = $"{"Hello world!"}""
+    End Sub
+End Module]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Dim formatt ... o world!"}"')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'formattable')
+    Variables: Local_1: formattable As System.IFormattable
+    Initializer: IConversionExpression (ConversionKind.InterpolatedString, Implicit) (OperationKind.ConversionExpression, Type: System.IFormattable) (Syntax: '$"{"Hello world!"}"')
+        IInterpolatedStringExpression (OperationKind.InterpolatedStringExpression, Type: System.String) (Syntax: '$"{"Hello world!"}"')
+          Parts(1): IInterpolation (OperationKind.Interpolation) (Syntax: '{"Hello world!"}')
+              Expression: ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: "Hello world!") (Syntax: '"Hello world!"')
+]]>.Value
```

Comment:
Yeah, we have open bugs for fixing up the formatting of the generated test expressions. They're low on the priority list for now though.